libfwupdplugin/fu-efivar-linux.c

```diff
@@ -329,6 +329,23 @@ fu_efivar_space_used_impl(GError **error)
 	guint64 total = 0;
 	g_autoptr(GDir) dir = NULL;
 	g_autofree gchar *path = fu_efivar_get_path();
+	g_autoptr(GFile) file_fs = g_file_new_for_path(path);
+	g_autoptr(GFileInfo) info_fs = NULL;
+	g_autoptr(GError) error_local = NULL;
+
+	/* this is only supported in new kernels */
+	info_fs = g_file_query_info(file_fs,
+				    G_FILE_ATTRIBUTE_FILESYSTEM_USED,
+				    G_FILE_QUERY_INFO_NONE,
+				    NULL,
+				    &error_local);
+	if (info_fs == NULL) {
+		g_debug("failed to get efivar used space: %s", error_local->message);
+	} else {
+		total = g_file_info_get_attribute_uint64(info_fs, G_FILE_ATTRIBUTE_FILESYSTEM_USED);
+		if (total > 0)
+			return total;
```

Comment:
at least until it's proved out, can we see if the numbers for adding each file and using the API are different?

Reply:
I think yes as a g_warning -- but I'm hesitant to record it to the LVFS otherwise we'll either need to return two numbers or have two APIs -- that need win32/macos stubs too. Was a g_warning what you thought?

Reply:
Done.